information-schema.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Information Schema
+summary: TiDB implements the ANSI-standard information_schema for viewing system metadata.
+category: reference
+aliases: ['/docs/dev/information-schema/information-schema-information-schema/','/docs/dev/reference/system-databases/information-schema/','/tidb/dev/system-table-information-schema/']
+---
+
+# Information Schema
+
+Information Schema provides an ANSI-standard way of viewing system metadata. TiDB also provides a number of custom `INFORMATION_SCHEMA` tables, in addition to the tables included for MySQL compatibility.
+
+Many `INFORMATION_SCHEMA` tables have a corresponding `SHOW` command. The benefit of querying `INFORMATION_SCHEMA` is that it is possible to join between tables.
+
+| Table Name                                                                              | TiDB Extension | Description |
+|-----------------------------------------------------------------------------------------|----------------|-------------|
```

Comment:
how about grouping all the tidb extension tables into one section to help users get a full view of these tables.

Reply:
Works for me. I've made this change.